providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/custom_ops.py

```diff
@@ -10,6 +10,12 @@
 logger = logging.getLogger("openfeature.contrib")
 
 
+class Fraction:
+    weight: int = 1
+    variant: str
+    pass
```

Comment:
```suggestion
@dataclass
class Fraction:
    variant: str
    weight: int = 1
```
please use a `dataclass` for such a case and fields with a default value go to the bottom